package.json

```diff
@@ -34,16 +34,17 @@
     "hot-server": "webpack serve --host 0.0.0.0 --config webpack/dev.babel.js",
     "open-static": "node -e 'require(\"open\")(\"http://localhost:3001\")'",
     "lint": "eslint --cache --ext '.js,.jsx' src test",
-    "lint-errors": "eslint --cache --quiet --ext '.js,.jsx' src test",
+    "lint-errors": "eslint --cache --no-error-on-unmatched-pattern --quiet --ext '.js,.jsx' src test",
     "lint-fix": "eslint --cache --ext '.js,.jsx' src test --fix",
     "cy:start": "webpack serve --port 3260 --host 0.0.0.0 --config webpack/dev.babel.js --hot --progress --static test/e2e/static",
     "cy:open": "cypress open",
     "cy:run:chrome": "cypress run --browser chrome",
     "cy:run:firefox": "cypress run --browser firefox",
     "cy:run:electron": "cypress run --browser electron",
     "cy:dev": "cross-env CYPRESS_BASE_URL=http://localhost:3260/ start-server-and-test cy:start http://localhost:3260 cy:open",
+    "cy:dev-e2e": "cross-env CYPRESS_BASE_URL=http://localhost:3260/ start-server-and-test cy:start http://localhost:3260 cy:run:electron",
```

Comment:
Generated a git patch for making this right.

Using `build:serve` limits us in what we can do with the fragments and how we can configure to run them in e2e tests. Instead I've configured webpack and babel to mimic building of production build fragments during the e2e testing in CI.